### PR TITLE
Now supports Universal Link

### PR DESCRIPTION
**Pull request by ivanfei-1**
You can jump to floor or hole using https://www.fduhole.com/hole/114514 or https://www.fduhole.com/floor/114514

Get unmounted error with `smartNavigatorPush()` and tests are requested on Android.

### CODE REVIEW

File: ios/Runner/Runner.entitlements [hunk 1]
```diff
@@ -6,7 +6,7 @@
 	<string>development</string>
 	<key>com.apple.developer.associated-domains</key>
 	<array>
-		<string>webcredentials:www.fduhole.com</string>
```
Comment (singularity-s0) [inline ios/Runner/Runner.entitlements:9]: Why remove `webcredentials`?

**Review by singularity-s0 — CHANGES_REQUESTED** [review]

File: ios/Runner/Runner.entitlements (hunk 1 above)
Comment (ivanfei-1) [reply ios/Runner/Runner.entitlements:9]: It's added back on commit 2abc554

**Comment by ivanfei-1** [comment]
Unmounted error resolved.

**Review by singularity-s0 — APPROVED** [review]

**Review by w568w — APPROVED** [review]

**Comment by w568w** [comment]
This should close #227.

**Pull request merged**